config/form_profile_mappings/22-5490.yml

```diff
@@ -1,5 +1,3 @@
-veteranDateOfBirth: [identity_information, date_of_birth]
-veteranFullName: [identity_information, full_name]
-veteranSocialSecurityNumber: [identity_information, ssn]
-toursOfDuty: [military_information, tours_of_duty]
```

Comment:
I went back and looked at the paper forms and it looks like it is asking for the applicant's military service. In which case, I think we can keep it here and on the 5495.